pc/protocols/usb_boot.h

```diff
@@ -21,6 +19,7 @@ extern int usb_loglevel;
 #define DEFAULT_UNBOOTVID           0x03E7
 #define DEFAULT_UNBOOTPID_2485      0x2485
 #define DEFAULT_UNBOOTPID_2150      0x2150
+#define DEFAULT_BOOTLOADER_PID      0xf63c
```

Comment:
We could use `0xB001`=BOOT as PID, it's nicer IMO.
It's [not taken](http://www.linux-usb.org/usb.ids), at least for 0x03E7 VID.

Reply:
Clever.  :-). Makes it very clear.